core/src/Dscp/Core/Fees.hs

```diff
@@ -102,3 +103,20 @@ fixLinearFees coeffs buildTx = go (Fees $ fcMinimal coeffs)
 fixFees :: FeePolicy Tx -> (Fees -> TxWitnessed) -> TxWitnessed
 fixFees feePolicy = case feePolicy of
     LinearFeePolicy coeffs -> fixLinearFees coeffs
+
+-- | Estimate fees for given outputs.
+-- Only works for fixed nonce, key and signature sizes.
+estimateFees :: FeeConfig -> [TxOut] -> Fees
+estimateFees feeConfig outs = calcFeeG feeConfig gTx
+  where
+    publicKey = leftToPanic $ fromByteArray ("patakbardaq_skovoroda_pvaforever" :: ByteString)
+    signature = leftToPanic $ fromByteArray ("patakbardaq_skovoroda_pvaforever_and_thirty_two_more_bytes_mkay?" :: ByteString)
```

Comment:
For both you can use `detGen 42 arbitrary` if you care about size which may change.

Reply:
well yes, it doesn't differ conceptually